src/nomad_simulations/schema_packages/model_system.py

```diff
@@ -1027,12 +955,50 @@ class ModelSystem(System):
         """,
     )
 
+    representations = SubSection(
+        sub_section=RelativeRepresentation.m_def,
+        repeats=True,
+        description="""
+        Alternative representations of the original model system, e.g., in a different simulation box or crystal cell.
+        """,
+    )
+
     sub_systems = SubSection(sub_section=SectionProxy('ModelSystem'), repeats=True)
 
+    # Backward compatibility: add cell as a subsection that references Representation
```

Comment:
imo we do not need to provide this at the development stage it will just create a lot of clutter.

Reply:
Sorry for the extraneous comment. No, `cell` is being removed as per the title of the PR.